ndcube/asdf/resources/schemas/ndcube-0.1.0.yaml

```diff
@@ -0,0 +1,25 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/NDCube-0.1.0"
+
+title:
+  Represents the ndcube NDCube object
+
+description:
+  Represents the ndcube NDCube object
+
+type: object
+properties:
+  data:
+    description: "Must be compatible with ASDF serialization/deserialization and supported by NDCube."
+  wcs:
+    tag: "tag:stsci.edu:gwcs/wcs-1.*"
+  extra_coords:
+    tag: "tag:sunpy.org:ndcube/extra_coords/extra_coords/extracoords-0.*"
+  global_coords:
+    tag: "tag:sunpy.org:ndcube/global_coords/globalcoords-0.*"
+
+required: [data, wcs]
+additionalProperties: true
```

Comment:
One question about this. What's the reasoning for having this `true` where it's `false` in the other schemas?

Reply:
We had discussed during the video chat that setting the `additionalProperties` to true would prevent conflicts when subclassing NDCube and writing the schema for the subclass.
Here is the related [comment](https://github.com/sunpy/ndcube/pull/708#:~:text=If%20I%20subclass%20NDCube%20and%20want%20to%20write%20my%20own%20schema%20for%20my%20subclass%20with%20additional%20properties%20does%20this%20get%20in%20the%20way%3F%20%40braingram)

Reply:
Thanks! I forgot about that conversation. Looks good to me!